tuner-project.json

```diff
@@ -0,0 +1 @@
ionState":1,"SteerEncoderSelectionState":1,"IsModuleValidationComplete":true,"ValidatedSteerId":3,"ValidatedDriveId":1,"ValidatedEncoderId":2},{"ModuleName":"Back Left","ModuleId":2,"Encoder":{"Id":3,"Name":"Rear Left","Model":"CANCoder","CANbus":"68F8EBC54C324353202020591C4117FF","CANbusFriendly":"Swerve Drive","SelectedMotorType":null,"IsStandaloneFx":false},"SteerMotor":{"Id":4,"Name":"Rear Left Turn","Model":"Talon FX vers. C","CANbus":"68F8EBC54C324353202020591C4117FF","CANbusFriendly":"Swerve Drive","SelectedMotorType":{"Name":"WCP Kraken x60","FreeSpeedRps":96.7,"SlipCurrentLimit":120,"StatorCurrentLimit":60},"IsStandaloneFx":false},"DriveMotor":{"Id":6,"Name":"Rear Left Drive","Model":"Talon FX vers. C","CANbus":"68F8EBC54C324353202020591C4117FF","CANbusFriendly":"Swerve Drive","SelectedMotorType":{"Name":"WCP Kraken x60","FreeSpeedRps":96.7,"SlipCurrentLimit":120,"StatorCurrentLimit":60},"IsStandaloneFx":false},"IsEncoderInverted":false,"IsSteerInverted":false,"SelectedEncoderType":"CANcoder","EncoderOffset":-0.124755859375,"DriveMotorSelectionState":1,"SteerMotorSelectionState":1,"SteerEncoderSelectionState":1,"IsModuleValidationComplete":true,"ValidatedSteerId":4,"ValidatedDriveId":6,"ValidatedEncoderId":3},{"ModuleName":"Back Right","ModuleId":3,"Encoder":{"Id":1,"Name":"Rear Right","Model":"CANCoder","CANbus":"68F8EBC54C324353202020591C4117FF","CANbusFriendly":"Swerve Drive","SelectedMotorType":null,"IsStandaloneFx":false},"SteerMotor":{"Id":7,"Name":"Rear Right Turn","Model":"Talon FX vers. C","CANbus":"68F8EBC54C324353202020591C4117FF","CANbusFriendly":"Swerve Drive","SelectedMotorType":{"Name":"WCP Kraken x60","FreeSpeedRps":96.7,"SlipCurrentLimit":120,"StatorCurrentLimit":60},"IsStandaloneFx":false},"DriveMotor":{"Id":0,"Name":"Rear Right Drive","Model":"Talon FX vers. C","CANbus":"68F8EBC54C324353202020591C4117FF","CANbusFriendly":"Swerve Drive","SelectedMotorType":{"Name":"WCP Kraken x60","FreeSpeedRps":96.7,"SlipCurrentLimit":120,"StatorCurrentLimit":60},"IsStandaloneFx":false},"IsEncoderInverted":false,"IsSteerInverted":false,"SelectedEncoderType":"CANcoder","EncoderOffset":0.33349609375,"DriveMotorSelectionState":1,"SteerMotorSelectionState":1,"SteerEncoderSelectionState":1,"IsModuleValidationComplete":true,"ValidatedSteerId":7,"ValidatedDriveId":0,"ValidatedEncoderId":1}],"SwerveOptions":{"kSpeedAt12Volts":5.761476779868646,"Gyro":{"Id":0,"Name":"Pigeon","Model":"Pigeon 2 vers. S","CANbus":"68F8EBC54C324353202020591C4117FF","CANbusFriendly":"Swerve Drive","SelectedMotorType":null,"IsStandaloneFx":false},"IsValidGyroCANbus":true,"VerticalTrackSizeInches":23.125,"HorizontalTrackSizeInches":23.125,"WheelRadiusInches":2.0,"IsLeftSideInverted":false,"IsRightSideInverted":true,"SwerveModuleType":0,"SwerveModuleConfiguration":{"ModuleBrand":-1,"DriveRatio":5.54,"SteerRatio":25.0,"CouplingRatio":0.0,"CustomName":null},"HasVerifiedSteer":true,"SelectedModuleManufacturer":"Custom","HasVerifiedDrive":true,"IsValidConfiguration":true}}
```

Comment:
@frc10101/students @frc10101/mentors this is the file that the tuner generates when you make a new project, or you can import this file and start from that if changes are needed